### PR TITLE
#30: Improved connection option documentation.

### LINKED ISSUE #30: Documentation: Recommendations
Additional Section in Documentation like
Recomendation for deployment
Hoe to choose the best configuration for RLS
rls against 
(1) local exasol (pointing to different schema): use is_local + exa
(2) remote exasol: use exa
(3) remote exasol with blocked ports: use jdbc
(4) remote non exasol: currently no rls possible

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #30.

Task-Url: http://github.com/exasol/row-level-security/issues/30

### CODE REVIEW

File: doc/user_guide/user_guide.md [hunk 1]
```diff
@@ -276,6 +296,41 @@ The minimum requirements for a regular user in order to be able to access the RL
...
+
+    IS_LOCAL = 'true'
+
+The parameter `IS_LOCAL` provides an additional speed-up in this particular use case. The way this works is that RLS generates a regular `SELECT` statement instead of an `IMPORT` statement. And that `SELECT` can be directly executed by the core database, whereas the `IMPORT` statement takes a detour via the ExaLoader.
+
+That means whenever you are protecting data on the same instance or cluster, you definitely should use `IS_LOCAL`!
+
+#### Data Source is an Exasol Instance or Cluster Only Reachable via JDBC
+
+Why this connection type works, it is also the slowest option and exists mainly to support integration tests on the ExaLoader. We recommend that you use `IMPORT FROM EXA` instead.
```
Comment (ghost) [inline doc/user_guide/user_guide.md]: "Why this connection type works" - what does this mean?
An important reason for this way of connecting is, that import_from_exa requires a wider range of ports that have to be open.

**Review by ghost — CHANGES_REQUESTED** [review]

File: doc/user_guide/user_guide.md (hunk 1 above)
Comment (redcatbear) [reply doc/user_guide/user_guide.md]: "Why" --> "While"

I asked about ports yesterday and according to the core team they are the same. Needless to say I was surprised.

Comment (ghost) [reply doc/user_guide/user_guide.md]: But I remember that I had problems ...

Comment (ghost) [reply doc/user_guide/user_guide.md]: but nevertheless I am fine now on doc side

**Review by ghost — APPROVED** [review]

**Pull request merged**